counter-sink/src/main/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+server:
+  port: 8081
+spring:
+  cloud:
+    stream:
+      bindings:
+        input: ticktock
```

Comment:
maybe "counter" or at least something not copied from a completely different context ;)